store/storage_manager_builder.go

```diff
@@ -118,17 +120,35 @@ func (smb *StorageManagerBuilder) Build(ctx context.Context) (*Manager, error) {
 		return nil, fmt.Errorf("dispersal backend is set to V1, but V1 backend is not enabled")
 	}
 
+	var kzgVerifier *kzgverifier.Verifier
+	// there are two cases in which we need to construct the kzgVerifier:
+	// 1. V1
+	// 2. V2, when validator retrieval is enabled
+	if v1Enabled || v2Enabled && slices.Contains(smb.v2ClientCfg.RetrieversToEnable, common.ValidatorRetrieverType) {
+		// The verifier doesn't support loading trailing g2 points from a separate file. If LoadG2Points is true, and
+		// the user is using a slimmed down g2 SRS file, the verifier will encounter an error while trying to load g2
+		// points. Since the verifier doesn't actually need g2 points, it's safe to force LoadG2Points to false, to
+		// sidestep the issue entirely.
+		kzgConfig := smb.kzgConfig
```

Comment:
lol more beautiful srs loading code. I have no idea what this means... hoping you figured it out and its not too much bad tech debt.

Reply:
Everything to do with the kzg usage is required to be held together by duct tape, unfortunately. It'll be the case until we buckle down and rework the underlying utilities. I don't think this is really adding to the tech debt: it's more just an expression of the existing debt